backend/server.js

```diff
@@ -16,6 +18,106 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const UserSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  accessToken: {
+    type: String,
+    default: () => crypto.randomBytes(128).toString("hex"),
+  },
+});
+
+const User = mongoose.model("User", UserSchema);
+
+app.post("/register", async (req, res) => {
+  const { username, password } = req.body;
+
+  try {
+    const salt = bcrypt.genSaltSync();
+
+    if (password.length < 8) {
+      res.status(400).json({
+        response: "Password must be at least 8 characters long",
+        success: false,
+      });
+    } else {
+      console.log(password);
+      console.log(username);
```

Comment:
Maybe you could remove these console.logs so the password and username don't get visiable